tools/tools.go

```diff
@@ -25,11 +25,14 @@ import (
 	_ "golang.org/x/tools/cmd/stringer"
 	_ "golang.org/x/vuln/cmd/govulncheck"
 	_ "mvdan.cc/gofumpt"
+
+	_ "github.com/FerretDB/FerretDB/tools/generatechangelog"
```

Comment:
It should not be there – note that checkdocs, checkswitch, checkcomments are all not imported

Reply:
Removed verified binary generation still works.